shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirImplementation.java

```diff
@@ -13,10 +13,22 @@ public class HapiFhirImplementation implements HapiFhir {
 
     private static final HapiFhirImplementation INSTANCE = new HapiFhirImplementation();
     private static final FhirContext CONTEXT = FhirContext.forR4();
-    private static final IFhirPath PATH_ENGINE = CONTEXT.newFhirPath();
+
+    private static final IFhirPath PATH_ENGINE = createEngine();
 
     private HapiFhirImplementation() {}
 
+    /**
+     * Creates FHIRPath engine with a custom evaluation context.
+     *
+     * @return Configured engine.
+     */
+    private static IFhirPath createEngine() {
+        var engine = CONTEXT.newFhirPath();
+        engine.setEvaluationContext(new HapiFhirCustomEvaluationContext());
+        return engine;
+    }
```

Comment:
Just a small thing, but mind moving this under method under the `getInstance` method?  `public static` methods should come before `private static` methods.